src/util/HttpServer/ContentEncodingHelper.h

```diff
@@ -48,4 +48,20 @@ inline void setContentEncodingHeaderForCompressionMethod(
   }
 }
 
+inline std::ostream& operator<<(std::ostream& out,
+                                CompressionMethod compressionMethod) {
+  switch (compressionMethod) {
+    case CompressionMethod::NONE:
+      out << "CompressionMethod::NONE";
+      break;
+    case CompressionMethod::DEFLATE:
+      out << "CompressionMethod::DEFLATE";
+      break;
+    case CompressionMethod::GZIP:
+      out << "CompressionMethod::GZIP";
+      break;
+  }
+  return out;
+}
+

```

Comment:
I mean if we ever plan to `cout << CompressionMethod::GZIP` or something this will come in handy, currently it is used in 3 tests and the implementation is rathert straightforward, so I think it should be kept here